inject.css

```diff
@@ -26,7 +26,7 @@
 /* e.g. https://www.igvita.com/2012/09/12/web-fonts-performance-making-pretty-fast/ */
 .html5-video-player:not(.ytp-hide-info-bar) .vsc-controller {
   position: relative;
-  top: 40px;
```

Comment:
Is there a way to target this to fullscreen mode only? Otherwise this offsets the controller on all embeds, which looks a bit wonky.

Reply:
ytp-hide-info-bar is only visible in normal screen on youtube.com
since they changed embed's style, we can also target embed with or without fullscreen. (channel logo on title bar)